src/ldw-barsPlus.js

```diff
@@ -197,14 +197,15 @@ export default {
     // Process two dimensional data
     g.nDims = 2;
 
-    var p1 = "", p2, edges = [], b, p = [];
+    var p1 = "", p2, edges = [], b, p = [] , l=[];
```

Comment:
What is `l` that is introduced here? Could you rename it, please? You don't have to rename all variables, but when we introduce a new one, I'd think it's better to be explicit.